src/widgets/tasks.rs

```diff
@@ -28,7 +28,8 @@ impl Tasks {
             sorted_tasks.sort();
             sorted_tasks.reverse();
 
-            for task in &sorted_tasks {
+            for task in &mut sorted_tasks {
+                task.subject = format!("{} - Test", task.subject);
```

Comment:
This probably should not have been commited.

Reply:
Correct I was just poking at things, didn't realize it had got committed! reverted the change here